admin_guide/quota.adoc

```diff
@@ -476,3 +476,48 @@ If a quota has been defined for your project, see
 xref:../dev_guide/deployments/basic_deployment_operations.adoc#deployment-resources[Deployment Resources]
 for considerations on any deployment configurations.
 endif::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+[[limited-resources-quota]]
+== Configuring resources whose consumption is limited by default
+
+[NOTE]
+====
+This feature is tech preview and subject to change in future releases.
+====
+
+Administrators may want to restrict consumption of high-cost compute or storage
+resources by default.  Traditionally, if a resource was not under quota, users
+were entitled to consume as much of that resource by default.  It is possible
```

Comment:
This paragraph is a bit confusing, again about what the out-of-box treatment of non-quota resources. Also, at this point, does the admin care how the feature works ("configure the quota system to identify...). The admin just wants to deny access. The how is explained later in better detail.  
"Administrators may want to restrict the consumption of high-cost compute or storage resources. By default, if a compute or storage resource is not covered by a quota, users can consume as much of that resource that is available. Administrators can configure {product-title} to deny the use of compute and storage resources that are not covered by quota."